main/menus.js

```diff
@@ -121,6 +122,16 @@ const applicationMenuTemplate = [
       {
         type: 'separator'
       },
+      {
+        label: 'Save Original…',
```

Comment:
It should have the Command+S accelerator.

Reply:
41c7479